packages/app/src/components/DBSearchPageFilters.tsx

```diff
@@ -549,6 +525,25 @@ export const FilterGroup = ({
             }}
           >
             <Stack gap={0}>
+              {/* Show search bar if expanded and there are more than 5 values */}
+              {isExpanded && augmentedOptions.length > 5 && (
+                <div className="px-2 pb-2">
+                  <TextInput
+                    size="xs"
+                    placeholder="Search values..."
+                    value={search}
+                    data-testid={`filter-search-${name}`}
+                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                      setSearch(event.currentTarget.value)
+                    }
+                    rightSectionWidth={20}
+                    rightSection={<IconSearch size={12} stroke={2} />}
+                    classNames={{
+                      input: 'ps-0.5',
+                    }}
+                  />
+                </div>
```

Comment:
Move search bar underneath filter and only show if there are 5 or more values. 5 was arbitrarily chosen, but the point is: its only useful when there are a lot of values to choose from. Worked this out with Elizabet.